bentoml/_internal/tracing/jaeger.py

```diff
@@ -2,7 +2,7 @@
 from contextlib import contextmanager
 from contextvars import ContextVar
 
-from bentoml._internal.tracing import Tracer
+from ..tracing import Tracer
```

Comment:
this can be just `from . import Tracer`?

Reply:
will fix it